src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -501,16 +501,26 @@ public function parents()
     /**
      * Returns the children nodes of the current selection.
      *
+     * @param $selector string|null A optional CSS selector to filter children
+     *
      * @return self
      *
      * @throws \InvalidArgumentException When current node is empty
+     * @throws \RuntimeException         if the CssSelector Component is not available and $selector is provided
      */
-    public function children()
+    public function children($selector = null)
```

Comment:
Actually, this is a BC break: child classes overriding this method would break with this change.
What we do in these situations is fetching the argument using \func_get_arg(), see e.g. the `Finder::sortByName()` method.
`public function children(/* string $selector = null */)` should be the new signature